fcrepo-transform/src/main/java/org/fcrepo/transform/TransformationFactory.java

```diff
@@ -56,7 +59,7 @@ public TransformationFactory() {
     public <T> Transformation<T> getTransform(final MediaType contentType, final InputStream inputStream) {
         final String mimeType = contentType.toString();
         if (mimeToTransform.containsKey(mimeType)) {
-            return (Transformation<T>) mimeToTransform.get(contentType.toString()).newTransform(inputStream);
+            return mimeToTransform.get(contentType.toString()).apply(inputStream);
```

Comment:
This seems incorrect to me. The previous logic returned a new transformation with the arg `inputStream` query. The new logic returns a new transformation trying to apply the arg `inputStream` as if it were an `RdfStream`.